packages/lib/apiKey/cache.ts

```diff
@@ -0,0 +1,34 @@
+import { revalidateTag } from "next/cache";
+
+interface RevalidateProps {
+  id?: string;
+  environmentId?: string;
+  apiKey?: string;
+}
+
+export const apiKeyCache = {
+  tag: {
+    byId(id: string) {
+      return `apiKeys-${id}`;
+    },
+    byEnvironmentId(environmentId: string) {
+      return `environments-${environmentId}-apiKeys`;
+    },
+    byApiKey(apiKey: string) {
```

Comment:
Please rename to `byHashedKey` to avoid confusion if this is the hashed or non-hashed key